src/style.css

```diff
@@ -0,0 +1,121 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.page {
+  box-sizing: border-box;
+  width: 100%;
+  border-radius: 0;
+  margin: 0 auto;
+  background: #fff;
+  position: relative;
+}
+
+.bigForm {
+  height: 70px;
+  margin: 20px 0;
+}
+
+.bigField {
+  position: relative;
+}
+
+.bigBar {
+  box-sizing: border-box;
+  width: 100%;
+  height: 100%;
+  background: #fff;
+  font-family: Avenir, sans-serif;
+  line-height: 22px;
+  font-size: 16px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  padding: 23px 0 23px 62px;
+}
+
+.bigBar::placeholder {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.bigBar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.bigBar:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  font-weight: 900;
+  text-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  outline: none;
+}
+
+.smallField {
+  position: relative;
+  height: 42px;
+  background: #fff;
+}
+
+.smallBar {
+  box-sizing: border-box;
```

Comment:
move `box-sizing` to the `*` selector or remove it

Reply:
fix this comment

Reply:
and remove other `box-sizing`